mix.exs

```diff
@@ -64,6 +64,9 @@ defmodule GoogleScraping.MixProject do
       {:phoenix_live_dashboard, "~> 0.6"},
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:phoenix_live_view, "~> 0.17.5"},
+      {:phoenix_pagination,
+       git: "https://github.com/FunkyStudioHQ/phoenix_pagination.git",
+       ref: "558de222d268529031de8782e8027c01831076af"},
```

Comment:
Is there a reason to using the Git version instead of `{:phoenix_pagination, "~> 0.7.0"}`? 🤔

Reply:
`0.7.0` can't be installed with modern Phoenix version: https://github.com/FunkyStudioHQ/phoenix_pagination/issues/7

They fixed it in main branch, but didn't release new version yet.